### ISSUE TITLE
Understand the difference between K3s and K3d

### ISSUE THREAD

**Issue by FirelightFlagboy** [Ragarnoy/Inception-of-Things#35]

**Comment by Ragarnoy** [comment]
https://thechief.io/c/editorial/k3s-vs-k3d/

**Comment by Eviber** [comment]
k3s is a production-grade Kubernetes server that is designed to be lightweight and easy to use. It includes a pre-selected and optimized set of Kubernetes features for use on lightweight machines or in development and test environments.

k3d is a Kubernetes cluster deployment tool that uses k3s to create and manage development and test clusters on a single machine. It offers a simple command-line interface for creating and deleting k3s clusters and running commands on them.

In summary, k3s is a full Kubernetes server designed to be lightweight and easy to use, while k3d is a Kubernetes cluster deployment tool that uses k3s to create and manage development and test clusters.
